pkgs/applications/emulators/pcsx-redux/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchFromGitHub, zlib, pkg-config, libuv, curl, ffmpeg, glfw, capstone, freetype, libX11 }:
```

Comment:
It seems one package per line is preferred.

Reply:
Sorry, I pushed out of date commit. Just pushed an update. I'll deal with other comments tomorrow, thank you.